vendor/k8s.io/kube-openapi/pkg/generators/openapi.go

```diff
@@ -336,6 +336,7 @@ func (g openAPITypeWriter) generate(t *types.Type) error {
 		}
 		g.Do("return $.OpenAPIDefinition|raw${\nSchema: spec.Schema{\nSchemaProps: spec.SchemaProps{\n", args)
 		g.generateDescription(t.CommentLines)
+		g.Do("Type: []string{\"object\"},\n", nil)
```

Comment:
@nikhita  Where does this change come from? Do we need that? It generates a huge diff in the api/openapi-spec/swagger.json I am (lacking a lot of context here) a bit uncomfortable with.

Reply:
If I read https://github.com/kubernetes/kubernetes/pull/72138#discussion_r258975988 correctly, this should have not really gotten in with the original commit. Can we remove that maybe?

Reply:
Looking into it, I bumped kube-openapi to the commit in the other PR and these changes are expected with a bump to that commit (see https://github.com/kubernetes/kube-openapi/compare/c59034cc13d587f5ef4e85ca0ade0c1866ae8e1d...ced9eb3070a5f1c548ef46e8dfe2a97c208d9f03). But I'll take a look if we can avoid bringing these in.

Reply:
Done. `pull-bazel-test` is currently failing on all 1.13 cherry-pick PRs. :/